buildpack.toml

```
@@ -110,6 +124,10 @@ api = "0.6"
     stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
     uri = "https://repo.spring.io/release/org/cloudfoundry/tomcat-access-logging-support/3.3.0.RELEASE/tomcat-access-logging-support-3.3.0.RELEASE.jar"
     version = "3.3.0"
+    purl = "pkg:generic/tomcat-access-logging-support@3.3.0"
+    cpes = [
+      ""
```

```suggestion
      "cpe:2.3:a:cloudfoundry:tomcat-access-logging-support:3.3.0:*:*:*:*:*:*:*"
```

Not sure there's a good answer on this one, but what about? Those are published under the CF Github org and there is actually a key for cloudfoundry.